lib/datadog/tracing/contrib/graphql/unified_trace.rb

```diff
@@ -0,0 +1,170 @@
+# frozen_string_literal: true
+
+require 'graphql/tracing'
+
+module Datadog
+  module Tracing
+    module Contrib
+      module GraphQL
+        # These methods will be called by the GraphQL runtime to trace the execution of queries
```

Comment:
Can you add a short blurb here about what exactly the `UnifiedTrace` represents, specially, if compared with the default `TracePatcher`?

I want to make sure we have enough context in the future to know whether we should switch to use this module by default, in the next major version. (actually, if it is desirable to switch to the `UnifiedTrace` by default, let's write that down in the comments here. We used the keyword `# DEV-3.0:` when looking into possible changes for the next major version upgrade, so we can do the same here, if it applies)

Reply:
Fixed !